pandas/tests/groupby/test_categorical.py

```diff
@@ -782,7 +782,7 @@ def test_categorical_no_compress():
 
 def test_sort():
 
-    # http://stackoverflow.com/questions/23814368/sorting-pandas-categorical-labels-after-groupby  # noqa: flake8
+    # http://stackoverflow.com/questions/23814368/sorting-pandas-categorical-labels-after-groupby  # noqa
```

Comment:
What was this change for?

Reply:
Somehow flake8 started whining about this line. It is an incorrect way of ignoring a line. I guess a flake8 update made this kind of indicating an ignoring illegal. So I fixed it to make flake8 pass. 

http://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html